mutt/dump-ical.py

```diff
@@ -57,7 +62,8 @@ def pretty_print_invitation(invitation):
     if invitees:
         print("Invitees:")
         for i in invitees:
-            print("\t{}".format(person_string(i)))
+            if 'CN' in i.params:
+                print("\t{}".format(person_string(i)))
```

Comment:
I confess that I put this together with a specific calendar server. It worked there and on the one used by my current employer but that's a small sample size so maybe I overlooked some nuance.

Would you mind elaborating on the circumstances this can happen?

Regardless, should we consider printing something else when there is no CN? Even if that ends up being a placeholder like `<unknown>`.

Reply:
I got the invite from Google Meet, no idea why there is no CN (maybe not in the address book?)